src/applications/disability-benefits/all-claims/content/form0781.jsx

```diff
@@ -0,0 +1 @@
+export const additionalFormsTitle = 'Additional Forms';
```

Comment:
I think you want the page's h3 title ('Additional forms to support your claim'), not the section/chapter title here.

Reply:
Agreed in call to keep the placeholder text. All user facing content has an update ticket for content. We are doing these later specifically to allow design to iron out any changes. This will be  corrected in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97071)

Reply:
I don't see where this is being imported.